templates/log-in-form.html

```diff
@@ -3,9 +3,9 @@
 % set email = state['log-in.email-sent-to']
 <div class="alert alert-success">
     <p>{{ _(
-        "We've sent you a single-use login link. Check your inbox, open the "
-        "provided link in a new tab, then come back to this page and click on "
-        "the button below to carry on with what you wanted to do."
+        "Check your inbox for a single-use login link, open the "
+        "link in it in a new tab, then come back to this page and click on "
```

Comment:
I am not sure this is really a concept that needs explaining. Users can handle tabs if they have them, otherwise they wont.